nmsg/io.c

```diff
@@ -715,6 +715,7 @@ io_write(struct nmsg_io_thr *iothr, struct nmsg_io_output *io_output,
 		return (res);
 
 	io_output->count_nmsg_payload_out += 1;
+	sync_fetch_and_add(&(io_output->count_nmsg_payload_out), 1);
```

Comment:
This appears to increment `io_output->count_nmsg_payload_out` again, increasing its value by 2.

Was this added line supposed to replace the previous line?

Reply:
My mistake, corrected.